css/properties/overflow-wrap.json

```diff
@@ -7,15 +7,15 @@
           "support": {
             "chrome": [
               {
-                "version_added": true
+                "version_added": "23"
               },
               {
                 "alternative_name": "word-wrap",
                 "version_added": "1"
               }
             ],
             "chrome_android": {
-              "version_added": null
+              "version_added": "70"
```

Comment:
I'm skeptical of the caniuse data here (and Opera and Opera for Android, below). That seems like a huge gap between shipping on desktop versus Android. I tried searching the Chromium source and wasn't able to learn anything interesting. Maybe @jpmedley can help us out here, to find out when `overflow-wrap` first landed?

This might also impact the `break-word` data below, too.

Reply:
Both should match Chrome.

Reply:
Thanks.